server/easy-deploy-nodepi3.sh

```diff
@@ -13,7 +13,7 @@ fi
 [ -z "$BRANCH" ] && BRANCH=master
 [ -z "$PROVIDERID" ] && PROVIDERID=""
 [ -z "$PROVIDERKEY" ] && PROVIDERKEY=""
-[ -z "$DAEMON_BIN_URL" ] && DAEMON_BIN_URL="https://lethean.io/lethean-cli-linux-arm-v3.1.tar.bz2"
+[ -z "$DAEMON_BIN_URL" ] && DAEMON_BIN_URL="https://download.lethean.io/lethean-cli-linux-arm-v3.1.tar.bz2"
```

Comment:
I am not sure this URL works, can we use the non-versioned URLs as they are uploaded?